public/js/test/mochitest/head.js

```diff
@@ -204,3 +221,27 @@ function isVisibleWithin(outerEl, innerEl) {
   return innerRect.top > outerRect.top &&
     innerRect.bottom < outerRect.bottom;
 }
+
+
+const selectors = {
+  callStackArrow: ".call-stack-pane .arrow",
+  frame: (index) => `.frames ul li:nth-child(${index})`
+}
```

Comment:
yeah - i've used this pattern a lot in other projects and is both concise and descriptive. I was struggling with differentiating UI and state methods like `getFrame`and I think this helps with that as well.